test/tests.js

```diff
@@ -47,47 +90,74 @@ describe("findBizDate(candidateDate, holidaysArray, direction)", () => {
 });
 
 describe("findNextBizDate(baseDate, holidaysArray, offset, direction)", () => {
-  it("Should be able to handle offset forward cases", () => {
+  it("Should be able to handle offset COUNTING forward cases", () => {
     let forwardCases = [
-      {baseDate: "2020-12-19", offset: 0 /* SAT 12/19  w*/, expectedResult: "2020-12-21"},
-      {baseDate: "2020-12-19", offset: 1 /* SUN 12/20  w*/, expectedResult: "2020-12-21"},
-      {baseDate: "2020-12-19", offset: 2 /* MON 12/21  b*/, expectedResult: "2020-12-21"},
-      {baseDate: "2020-12-19", offset: 3 /* TUE 12/22  b*/, expectedResult: "2020-12-22"},
-      {baseDate: "2020-12-19", offset: 4 /* WED 12/23  b*/, expectedResult: "2020-12-23"},
-      {baseDate: "2020-12-19", offset: 5 /* THU 12/24  b*/, expectedResult: "2020-12-24"},
-      {baseDate: "2020-12-19", offset: 6 /* FRI 12/25  h*/, expectedResult: "2020-12-29"},
-      {baseDate: "2020-12-19", offset: 7 /* SAT 12/26  w*/, expectedResult: "2020-12-29"},
-      {baseDate: "2020-12-19", offset: 8 /* SUN 12/27  w*/, expectedResult: "2020-12-29"},
-      {baseDate: "2020-12-19", offset: 9 /* MON 12/28  h*/, expectedResult: "2020-12-29"},
-      {baseDate: "2020-12-19", offset: 10 /*TUE 12/29  b*/, expectedResult: "2020-12-29"}
+      {baseDate: "2020-12-17", offset: 0 /*  0                     */, expectedResult: "2020-12-17"},
+      {baseDate: "2020-12-17", offset: 1 /*  0HWW1                 */, expectedResult: "2020-12-21"},
+
+      {baseDate: "2020-12-21", offset: 0 /*  0                     */, expectedResult: "2020-12-21"},
+      {baseDate: "2020-12-21", offset: 1 /*  01                    */, expectedResult: "2020-12-22"},
+      {baseDate: "2020-12-21", offset: 2 /*  012                   */, expectedResult: "2020-12-23"},
+
+      {baseDate: "2020-12-19", offset: 0 /*  WW0                   */, expectedResult: "2020-12-21"},
+      {baseDate: "2020-12-19", offset: 1 /*  WW1                   */, expectedResult: "2020-12-21"},
```

Comment:
Would it be better to not allow zero instead of having different logic when you pass zero?